index-ecology.md

```diff
@@ -30,9 +51,8 @@ We are passionate about training. So we are working in close collaboration with
 
 We then developed [several tutorials](https://galaxyproject.github.io/training-material/topics/ecology/){:target="_blank"} and more will come:
 
-- 
-- 
-- 
+- Introduction to species phenology study and abundance index as trends computation
```

Comment:
I guess we will add links later?

Reply:
yes, almost ready but need some finishing and reviews... Hope this will be ok for the end of the week!